data/ecosystems/n/nnsdao.toml

```diff
@@ -1,106 +1,63 @@
-# Ecosystem Level Information
```

Comment:
i think all those duplicates arise because we have different naming conventions. 
to create a new ecosystem, i take the github account and check if a toml file already exists for it. I also take the name of the github org as a title and search the data for duplicates (if no name is available, i simple take the github account). 

you on the other hand seem to give out ecosystem titles by hand or based on the github orgs name (?) and have a different naming convention for the toml files themselves (also based in the orgs name?). 

if you could give me a guide on how you decide for filename and ecosystem title i think we could avoid some of the issues here.

this particular issue is an example of what i mention above. i take the github account to create the `toml` file (`nnsdao`) , you seem to take the name of the github org (`nnsdao labs`).

Reply:
ideally we can do as discussed after this PR and hand out the search terms that you can then add to your internal tooling (if they are good enough) 😄

Reply:
I kinda like your idea of naming the ecosystem after the git organization name. However, an ecosystem is more nuanced than a simple git organization because it can have more than one org or it can simply not have orgs at all and has to be asigned an arbitrary name which could clash in the future with an org name.

Also, some orgs are named differently than what the ecosystem should be named, maybe because they changed name after the org creation or simply for no reason.

This is something we have to define well and make backwards compatible. Until then, we name the org after the most proper version of the project name. Currently we internally simply look for similar ecosystem names to make sure we are not creating a duplicate, or instead look for the org link and see if it's already linked to an ecosystem. I suggest you do that for now and get rid of the duplicates.

Reply:
That's a good question. There's this concept of inter-ecosystem relationship. It is still not documented nor reflected in the data files of this repo but we have plans to add it (soon). For many of our charts, the repo activity would count towards the ecosystem(s) it belongs to and also to the subsequent parent ecosystems to which the first parent has a **primary** relation with.

So, for your example, it would count towards B and C for sure if they both have both repos, and it would count towards A if any of B or C have a primary relation with A.

From the [official](https://www.developerreport.com/about) website:
<img width="639" alt="image" src="https://github.com/electric-capital/crypto-ecosystems/assets/5723039/afc990bd-a260-4f3d-bde7-4fcd2c8658af">